src/diffusers/models/unet_2d_blocks.py

```diff
@@ -962,6 +963,9 @@ def custom_forward(*inputs):
 
             output_states = output_states + (hidden_states,)
 
+        if additional_residuals is not None:
+            hidden_states += additional_residuals
```

Comment:
@williamberman will say again that this is incorrect, see my comment in original PR how to change this

Reply:
hey @bonlime I must have missed your comment on the original PR! could you link me?

Reply:
fixed! thanks for the heads up :)

Reply:
@williamberman actually, it looks the implementation was correct, since here it's changed inplace using `+=` in my local implementation it's written using `hidden_states = hidden_states + additional_residuals` and such implementation requires additionally changing the `output_states`. if you use `+=` it's not needed and probably even leads to doubly addition